service/src/main/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelper.java

```diff
@@ -24,7 +25,19 @@ public static String formatJudgeTitleAndName(JudgeAndLegalAdvisor judgeAndLegalA
             .orElse("");
     }
 
+    public static String formatJudgeTitleAndNameForDraftSDO(JudgeAndLegalAdvisor judgeAndLegalAdvisor,
```

Comment:
This shouldn't be died to SDO I think, better would be if method suggest why it's different behaviour. 

What if we would like to use it in another place?

Reply:
actually a redundant method - removed 👍